src/components/sheet/sheet-action-buttons/SendActionButton.js

```diff
@@ -1,24 +1,19 @@
 import lang from 'i18n-js';
 import React, { useCallback } from 'react';
 import SheetActionButton from './SheetActionButton';
-import { useExpandedStateNavigation } from '@/hooks';
 import Routes from '@/navigation/routesNames';
 import { IS_IOS } from '@/env';
+import { useNavigation } from '@/navigation';
 
 function SendActionButton({ asset, color: givenColor, ...props }) {
   const { colors } = useTheme();
   const color = givenColor || colors.paleBlue;
-  const navigate = useExpandedStateNavigation(null, false, asset);
```

Comment:
same note here: we had logic in the 1useExpandedStateNavigation1 that would also check if the wallet was 1readOnly1 and throw an alert if so